### PR TITLE
Initial code changes to create a salesforce-partners gem based out of omniauth-salesforce

### PR DESCRIPTION
**Pull request by vijayanands**

### CODE REVIEW

File: lib/omniauth-salesforce-partners/version.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module OmniAuth
+  module Salesforce
+    module Partners
```
Comment (trungpham) [inline lib/omniauth-salesforce-partners/version.rb]: Should not be namespacing this. Just call it SalesforcePartners.

File: lib/omniauth/strategies/salesforce-partners.rb [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 module OmniAuth
   module Strategies
-    class Salesforce < OmniAuth::Strategies::OAuth2
+    class Salesforce::Partners < OmniAuth::Strategies::OAuth2
```
Comment (trungpham) [inline lib/omniauth/strategies/salesforce-partners.rb]: Don't namespace

File: lib/omniauth/strategies/salesforce-partners.rb [hunk 3]
```diff
@@ -82,15 +82,15 @@ def raw_info
 
     end
 
-    class SalesforceSandbox < OmniAuth::Strategies::Salesforce
+    class SalesforceSandbox < OmniAuth::Strategies::Salesforce::Partners
```
Comment (trungpham) [inline lib/omniauth/strategies/salesforce-partners.rb]: no namespace

File: omniauth-salesforce-partners.gemspec [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
+  gem.homepage      = "https://github.com/realdoug/omniauth-salesforce-partners"
 
   gem.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   gem.files         = `git ls-files`.split("\n")
   gem.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
-  gem.name          = "omniauth-salesforce"
+  gem.name          = "omniauth-salesforce-partners"
   gem.require_paths = ["lib"]
-  gem.version       = OmniAuth::Salesforce::VERSION
+  gem.version       = OmniAuth::Salesforce::Partners::VERSION
```
Comment (trungpham) [inline omniauth-salesforce-partners.gemspec]: no namespace

File: spec/omniauth/strategies/salesforce_spec.rb [hunk 5]
```diff
@@ -1,12 +1,12 @@
 require 'spec_helper'
 
-describe OmniAuth::Strategies::Salesforce do
+describe OmniAuth::Strategies::Salesforce::Partners do
```
Comment (trungpham) [inline spec/omniauth/strategies/salesforce_spec.rb]: no namespace

**Comment by vijayanands** [comment]
@trungpham  : I have made the changes to remove the namespace. Can you please take a look at the changes and merge

**Pull request merged**